src/ray/protobuf/serialization.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package ray.serialization;
+
+message PythonBuffer {
+  uint64 address = 1;
+  int64 length = 2;
+  int64 itemsize = 3;
+  int32 ndim = 4;
+  bool readonly = 5;
+  string format = 6;
+  repeated int64 shape = 7;
+  repeated int64 strides = 8;
+  // 'suboffsets' is ignored since it is required to be NULL by pickle5 protocol.
+}
+
+message PythonObject {
+  uint64 inband_data_offset = 1;
```

Comment:
Can you also document all of these fields (and specify which buffer locations the offsets are relative to)?